components/error-view.js

```diff
@@ -4,17 +4,21 @@ const sendToAddon = require('../client-lib/send-to-addon');
 const sendMetricsEvent = require('../client-lib/send-metrics-event');
 const GeneralControls = require('./general-controls');
 
-module.exports = React.createClass({
-  getInitialState: function() {
-    return {hovered: false};
-  },
-  enterView: function() {
+module.exports = class ErrorView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {hovered: false};
```

Comment:
It has no effect on the actual dom's state. It's just initializing the state object with a value. The same way I would initialize a variable representing hover state in a vanilla js project.

``` js
var textHovered = false;
document.querySelector('.text').addEventListener('mouseenter', (ev) => textHovered = true);
document.querySelector('.text').addEventListener('mouseleave', (ev) => textHovered = false);
```